cfgov/cfgov/test.py

```diff
@@ -183,9 +184,12 @@ def assertPageIncludesHtml(self, page, s):
         request = RequestFactory().get('/')
         request.user = Mock()
 
-        rendered_html = page.serve(request).render()
+        rendered_html = page.serve(request).render().content
+        if six.PY3:
```

Comment:
Why is this extra step needed? Is it because the regexes [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/test.py#L173) aren't bytestrings?

Reply:
That's part of it, but there are places where we try to pass strings to those regexes as well.

If we want `assertHtmlRegexpMatches` to just expect to take `bytes` instead of `str` that's fine, it's just a different change.

Reply:
Other thought, I'm not sure why we're using a custom `assertPageIncludesHtml` instead of the Django `assertInHTML`